anchore_engine/services/policy_engine/engine/feeds/sync.py

```diff
@@ -120,6 +120,72 @@ def get_selected_feeds_to_sync(config):
         return [VulnerabilityFeed.__feed_name__, NvdV2Feed.__feed_name__]
 
 
+def get_selected_feeds_to_sync(config):
+    """
+    Returns the feeds to be synced based on the configuration.
+    Handles both legacy (top-level) and vulnerabilities provider (policy-engine) configurations
+
+    :param config: dict that is the system configuration
+    :return: list of strings of feed names to sync
+    """
+
+    # check for the vulnerabilities provider section first
+    provider_config = (
+        config.get("services", {}).get("policy_engine", {}).get("vulnerabilities", {})
+    )
+
+    if provider_config:  # new config added for grype integration found
+        logger.debug("Found a vulnerabilities provider configuration")
+        return get_feeds_to_sync_for_vulnerabilities_provider(provider_config)
+    else:  # fall back to older configuration
+        logger.debug("Falling back to legacy feeds configuration")
+        return get_selected_feeds_to_sync_legacy(config)
+
+
+def get_feeds_to_sync_for_vulnerabilities_provider(provider_config):
+    """
+    Expects a vulnerability provider config dictionary and returns the list of data feeds to sync
+
+    Example input:
+    {
+        "provider": "grype",
+        "feeds": {
+            "sync_enabled": True,
+            "ssl_verify": True,
+            "connection_timeout_seconds": 3,
+            "read_timeout_seconds": 60,
+            "data": {
+                "grypedb": {
+                    "enabled": True,
+                    "url": "https://toolbox-data.anchore.io/grype/databases/listing.json",
+                },
+                "packages": {
+                    "enabled": False,
+                    "anonymous_user_username": "anon@ancho.re",
+                    "anonymous_user_password": "pbiU2RYZ2XrmYQ",
+                    "url": "https://ancho.re/v1/service/feeds",
+                    "client_url": "https://ancho.re/v1/account/users",
+                    "token_url": "https://ancho.re/oauth/token",
+                },
+            },
+        },
+    }
+
+    Returns [grypedb]
+
+    """
+    logger.debug("Searching for data feeds to be synced in {}".format(provider_config))
```

Comment:
Should the logging use the percent format that @nightfurys  sent us: 

```
logger.debug("Searching for data feeds to be synced in %s" % provider_config)
```

Reply:
I think this was a copy paste of my own code and I still tend to use .format() out of habit. We can clear these up before the merge into master

Reply:
I'm not sure why we there is still a mix of various different formats for logging being discussed. Using `%` or `.format()` should not be preferred *for logging purposes* as [described here](https://github.com/anchore/enterprise/issues/150). To reiterate a bit the message:

> It is safer, and recommended, to not format string directly like that because it can cause applications to break when variables can be substituted. Logging offers the following solution:

```python
logger.info('some statement about %s', 'app')
```

> If string substitution doesn't work, the logging module will _log_ an error, showing what failed, preventing application breakage.

Reply:
This is important to understand :point_up:

Reply:
@alfredodeza there's no discussion about multiple formats. I shared the safe logging format with the grype-integration team but accidentally left the % operator for substitution instead of separate arguments. Thank you for explaining the format again!